servers/rendering_server.cpp

```diff
@@ -340,7 +340,7 @@ void _get_axis_angle(const Vector3 &p_normal, const Vector4 &p_tangent, float &r
 	if (d < 0.0) {
 		r_angle = CLAMP((1.0 - r_angle / Math_PI) * 0.5, 0.0, 0.49999);
 	} else {
-		r_angle = (r_angle / Math_PI) * 0.5 + 0.5;
+		r_angle = CLAMP((r_angle / Math_PI) * 0.5 + 0.5, 0.500008, 1.0);
```

Comment:
Out of curiosity, where does that `0.500008` come from? It might be worth adding a comment.

Reply:
Its the smallest number representable by a 16 bit UNORM value in the range 0.5 - 1.0.

Its the flipside of `0.49999` above.